tests/test_2fa_duo.py

```diff
@@ -206,15 +207,23 @@ def test_remember_me_cookie(
         # shib_idp_session, shib_idp_session_ss
 
         cookie_names = {cookie['name'] for cookie in fresh_browser.get_cookies()}
-        assert 'shib_idp_session' in cookie_names
+        # prod and eval are different for about a week til the new shib is released to prod.
+        if test_env == 'eval':
```

Comment:
I'm not familiar with the `uw-idp-web-tests`, but syntax-wise, this looks good to me. Do we have a Github issue to remove the if statements when the new Shibboleth is released to production? Other than that, it looks good to me. Let's see what the other folks think.

Thanks @goulter

Reply:
Thanks @soufianerafik , I created that issue [here](https://github.com/UWIT-IAM/uw-idp-web-tests/issues/53)